src/applications/gi/tests/e2e/cypress-helpers.js

```diff
@@ -0,0 +1,48 @@
+import calculatorConstantsJson from '../data/calculator-constants.json';
+import autocomplete from '../data/autocomplete.json';
+import institutionProfile from '../data/institution-profile.json';
+import searchResults from '../data/search-results.json';
+import { FORCE_OPTION } from 'platform/testing/e2e/cypress/support/form-tester';
+
+Cypress.on('uncaught:exception', (_err, _runnable) => {
```

Comment:
There were errors like below occurring and googling said solution was to add this which fixed the issue
```
     TypeError: Cannot read property 'run' of undefined
      at Context.eval (http://localhost:3001/__cypress/tests?p=src/platform/testing/e2e/cypress/support/index.js:24606:442975)
```